app/src/main/java/com/zarinraim/dogsearch/App.kt

```diff
@@ -0,0 +1,19 @@
+package com.zarinraim.dogsearch
+
+import android.app.Application
+import com.zarinraim.dogsearch.di.appModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.startKoin
+
+class App : Application() {
```

Comment:
It doesn't really matter right now, but usually, we give a name to the application class according to what the app is about. So, in your case, it could be DogBreedsApp or DogsApp or something like that.

Reply:
Changed to DogSearchApp as I named it in the beginning.